src/detect-filestore.c

```diff
@@ -224,7 +224,7 @@ static int DetectFilestorePostMatch(DetectEngineThreadCtx *det_ctx,
     else
         flags |= STREAM_TOSERVER;
 
-    for (uint16_t u = 0; u < det_ctx->filestore_cnt; u++) {
+    for (uint8_t u = 0; u < det_ctx->filestore_cnt; u++) {
```

Comment:
`det_ctx->filestore_cnt` is `uint16_t`, does this change make sense?

Reply:
This PR changes it to uint8_t in detect.h DetectEngineThreadCtx_ definition precisely because `DETECT_FILESTORE_MAX` (size of filestore array whose `filestore_cnt` is an index) is lesser than 256